jsoc/gsoc/medpipe3d.md

```diff
@@ -0,0 +1,82 @@
+# MedPipe3D Projects 
+
+MedPipe3D.jl together with MedEye3D.jl MedEval3D.jl and currently in development MedImage.jl is a set of libraries created to provide essential tools for 3D medical imaging to the Julia language ecosystem. Recently all of the projects migrated also to the JuliaHealth organization to improve packages visibility and promote interoperability.

```

Comment:
Some comments on this piece: 

1. Could you link the repositories here for each package you mentioned?
2. Could you make a markdown list to explain what (in just a few words) what each package does generally?
3. I'd remove the line about package migration.